src/components/MessageInput/MessageInput.tsx

```diff
@@ -487,6 +489,9 @@ export const MessageInputWithContext = <
                     />
                   ) : (
                     <>
```

Comment:
I think this needs to be more thought out, perhaps we should allow them to replace the entire attachment / command button section instead then go from there? We have been avoiding putting in these tunnels to add components everywhere else and instead ask users to replace components. Right now we are losing the show more functionality if they have this and commands but no images. And I am not sure how this will work if image selector is open and you open a custom one as we are only passing a function in, if you call that function and it's open it closes, but if it's closed it opens. I am also not sure the naming conveys the purpose and location here.

Reply:
So to summarize:
- Make InputOptions to replace attachment & commands button
- Rename InputOptions to something else. Any suggestions? InputOptions seems fine to me tbh.

> andd I am not sure how this will work if image selector is open

I will check it :)

Thanks :)

Reply:
InputButtons might be better, I wasn't sure what options was until I got down to the file, so just not semantic enough I think. But yeah I think it's better to make and replace just to follow the pattern and then it will be easier to deal with the collapsed mode internally there I think.

Reply:
Okies. `InputButtons` it is :)